### DISCUSSION TITLE
shutdown Ok, Cancel text line

### DISCUSSION THREAD

**Discussion by Leemann07** [OnionUI/Onion#1340] [Q&A]
![MainUI_000 copy](https://github.com/OnionUI/Onion/assets/152375478/7cec1b04-6ced-49b0-9f3f-965adbb7e74a)

I want to align the lines between A and text OK and B and Cancel.
How can i fix it ?

**Comment by Aemiii91** [comment]
Not possible. Unless you find the position in a hex editor (MainUI is closed source)

**Reply by Leemann07** [reply]
Thank you!!!

**Reply by REGoman** [reply]
Is it impossible to change the font color?

**Reply by Aemiii91** [reply]
@REGoman yes, also hardcoded
